src/features/book/bookSlice.js

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from '@reduxjs/toolkit';
+
```

Comment:
- [ ] Good job @kwesiObuobi, however, Kindly make sure to define `action` types for adding and removing a book as it is [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Define%20action%20types%20for%20adding%20and%20removing%20a%20book.). 👍, it'll be something like : 
 ````bash
const ADD_BOOK = 'bookStore/books/ADD_BOOK';
const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';

````

Reply:
Ohh Okay. I think I see the confusion. Please allow me to share my project requirements with you.
https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md
![requirements](https://user-images.githubusercontent.com/33374570/236875847-4f8a2e30-2884-48f3-b781-3c1df3b99735.png)

Reply:
So, that's our requirements this time, and I'm confident I have met all of them. Everyone else did the same and got their projects approved... And that's only because we weren't required to define action types since we're using RTK. Please am I still missing something?

Reply:
@topeogunleye

Reply:
Answered on slack ✔️